README.md

```diff
@@ -16,21 +23,25 @@ Here at Zendar, we rely on a fork of this repo.
 ### Rolling out updates at Zendar:
 
 If you need to update this code, the steps to do so are:
-1. make the code changes required, here in `ZendarInc/torchsparse`
+1. make the code changes required, here in `ZendarInc/torchsparseplusplus`
```

Comment:
I believe this change isn't correct - the repository is still named `torchsparse`.